examples/utils/get-tinylicious-container/src/insecureTinyliciousTokenProvider.ts

```diff
@@ -13,32 +13,32 @@ import { v4 as uuid } from "uuid";
  * to get up and running.
  */
 export class InsecureTinyliciousTokenProvider implements ITokenProvider {
-    constructor(private readonly documentId: string) {
-
-    }
-
-    public async fetchOrdererToken(): Promise<ITokenResponse> {
+    public async fetchOrdererToken(tenantId: string, documentId: string): Promise<ITokenResponse> {
         return {
             fromCache: true,
-            jwt: this.getSignedToken(),
+            jwt: this.getSignedToken(tenantId, documentId),
         };
     }
 
-    public async fetchStorageToken(): Promise<ITokenResponse> {
+    public async fetchStorageToken(tenantId: string, documentId: string): Promise<ITokenResponse> {
         return {
             fromCache: true,
-            jwt: this.getSignedToken(),
+            jwt: this.getSignedToken(tenantId, documentId),
         };
     }
 
-    private getSignedToken(lifetime: number = 60 * 60, ver: string = "1.0"): string {
+    private getSignedToken(
+        tenantId: string,
+        documentId: string,
+        lifetime: number = 60 * 60,
+        ver: string = "1.0"): string {
         // Current time in seconds
         const now = Math.round((new Date()).getTime() / 1000);
 
         const claims: ITokenClaims = {
-            documentId: this.documentId,
+            documentId,
             scopes: [ScopeType.DocRead, ScopeType.DocWrite, ScopeType.SummaryWrite],
-            tenantId: "tinylicious",
+            tenantId,
```

Comment:
Does tinylicious require the tenantId to be "tinylicious"? Regardless it's probably worth documenting somewhere for this class that this value must be provided for the tenantId param, or ignore the provided value and always use "tinylicious" (and document it gets ignored).  Mainly because the implementation before this never allowed you to provide the tenantId and it doesn't look like there are currently any examples for this change so it seems like this info might get lost

Reply:
the tinylicious url resolver hard codes the tenantId as tinylicious, which is how it should be, as the resolver gets the constituent parts from the url